securedrop/utils.py

```diff
@@ -0,0 +1,23 @@
+from itertools import count
+from logging import Logger
+from typing import Callable
+
+from jinja2.runtime import Context
+from jinja2.utils import pass_context
+
+
+# Note: return value type hint ignored because the returned function does not take in any
+# arguments, the pass_context decorator handles the Context argument automatically.
+def create_get_descriptive_title_function(logger: Logger) -> Callable[[], str]:  # type: ignore[return-value] # noqa: E501
```

Comment:
```suggestion
def create_get_descriptive_title_function(logger: Logger) -> Callable[[], str]:  # type: ignore [return-value] # noqa: E501
```

This should make **mypy** happy.

Reply:
Example in action: https://github.com/freedomofpress/securedrop-client/blob/main/securedrop_client/gui/widgets.py#L673